scripts/zones/Yuhtunga_Jungle/mobs/Mischievous_Micholas.lua

```diff
@@ -7,7 +7,7 @@ require("scripts/globals/regimes")
 -----------------------------------
 
 function onMobDeath(mob, player, isKiller)
-    tpz.hunts.checkHunt(mob, player, isKiller)
+    tpz.hunts.checkHunt(mob, player, 362)
```

Comment:
Seeing:
```lua
tpz.hunts.checkHunt(mob, player, 362)
tpz.regime.checkRegime(player, mob, 126, 1, tpz.regime.type.FIELDS)
```
Makes me want to say that we should establish a standardized argument order at some point to avoid inadvertently crossing wires. "Such and such arguments always come before this other argument". There's a lot of combat functions which waffle between `attacker, defender` and `defender, attacker` too.

Reply:
Personal opinion: checkHunt does it better visually because of the order onMobDeath uses. Just “feels right”.

Reply:
Indeed, I presume why @Apocarypse structured it so originally. :+1: